src/runtime/README.md

```diff
@@ -76,13 +38,20 @@ for details on the Kata Containers design.
 ## Configuration
 
 The runtime uses a TOML format configuration file called `configuration.toml`.
-The file contains comments explaining all options.
+The file is divided into sections for settings related to various
+parts of the system including the runtime itself, the [agent](../agent) and
+the [hypervisor](#hypervisor-specific-configuration).
 
-> **Note:**
+Each option has a comment explaining it use.
+
+> **Notes:**
+>
+> - The initial values in the configuration file provide a good default configuration.
+>   You may need to modify this file to optimise or tailor your system, or if you have
+>   specific requirements.
 >
-> The initial values in the configuration file provide a good default configuration.
-> You may need to modify this file to optimise or tailor your system, or if you have
-> specific requirements.
+> - The runtime will read the configuration file from one of [two
+>   possible locations](#stateless-systems).
```

Comment:
What about the `/opt/kata` tree?

Reply:
Yeah, there's a bit more detail here, indeed. it'd be more accurate to say what it will do by default. 

If we want to provide more detail, we can find a nice way to describe what actually happens, hopefully more succinctly than I do in my blather that follows:

It'll first check if a config path is specified in annotations associated with the pause container (yuck - I hope no one ever does this),  and then it'll check for a path specified within the CreateTaskRequest (this is what is done in the case of kata-deploy install, since we look at /opt/kata...), and then finally check the two default locations on the host filesystem, with /etc/kata-containers taking precedence over /usr/share/defaults/kata-containers....).  

See: https://github.com/kata-containers/kata-containers/blob/88a70d32ba5d3833ae84c8f65dd89fa17e3d0b1b/src/runtime/pkg/containerd-shim-v2/create.go#L212-L256